src/status_im/contexts/settings/wallet/keypairs_and_accounts/view.cljs

```diff
@@ -108,15 +112,15 @@
         :accessibility-label :keypairs-and-accounts-header
         :customization-color customization-color}]]
      [rn/view {:style style/settings-keypairs-container}
-      (when (seq missing-keypairs)
-        [quo/missing-keypairs
-         {:blur?            true
-          :keypairs         missing-keypairs
-          :container-style  style/missing-keypairs-container-style
-          :on-options-press on-missing-keypair-options-press}])
       [rn/flat-list
        {:data                    operable-keypairs
         :render-fn               keypair
+        :header                  (when (seq missing-keypairs)
+                                   [quo/missing-keypairs
+                                    {:blur?            true
+                                     :keypairs         missing-keypairs
+                                     :container-style  style/missing-keypairs-container-style
+                                     :on-options-press on-missing-keypair-options-press}])
```

Comment:
Here we're tweaking the keypairs and accounts lists to display the missing keypairs as an header. This causes the missing keypairs to scroll with the list.